apps/frontend/src/components/TimeCardPage/CellTypes/CommentCell.tsx

```diff
@@ -12,15 +12,29 @@ import ShowReportModal from "./CommentModals/ShowReportModal";
 interface CommentProps {
   comments: CommentSchema[] | undefined;
   date: number;
+  updateComments: Function;
   timesheetID: number;
 }
 
-export function CommentCell({ comments, date, timesheetID }: CommentProps) {
+export function CommentCell({
+  comments,
+  date,
+  updateComments,
+  timesheetID,
+}: CommentProps) {
   const [currentComments, setCurrentComments] = useState(
     getAllActiveCommentsOfType(CommentType.Comment, comments)
   );
   const [reports, setReports] = useState(
-    getAllActiveCommentsOfType(CommentType.Report, comments) as ReportSchema[]
+    getAllActiveCommentsOfType(CommentType.Report, comments).map((comment) => ({
+      AuthorID: comment.AuthorID,
+      Type: comment.Type,
+      Content: comment.Content.split(",")[0],
```

Comment:
I'd be careful using ',' as a delimiter here, since it wouldn't surprise me for the `Explanation` to include commas. Could you use something less frequently use to join and split the content, even just a semi-colon?